### ISSUE TITLE
Upload of raster time series

### ISSUE THREAD

**Issue by fzadrazil** [hslayers/hslayers-ng#3755]
# Feature Request

Layman supports publishing time series of rasters since version 1.18. Rasters should be uploaded in a zip package and a time regex must be provided so Layman knows where to look for the data/time in the file names, eg. 

`time_regex=[0-9]{8}` would look for the set of eight numbers, yyyyMMdd by default.
If the format is specified as part of the regex parameter, eg. `[0-9]{8},format=yyyyddMM`, it would be interpreted based on this format.

**Additional context**
[Add any other context or screenshots about the feature request here.
](https://github.com/LayerManager/layman/blob/master/doc/rest.md#post-workspace-layers:~:text=array%20of%20strings-,it%20is%20allowed%20to%20publish%20time%2Dseries%20layer,-by%20setting%20time_regex)

https://docs.geoserver.org/stable/en/user/community/cog/mosaic.html?highlight=app%20schema#timeregex-properties

**Comment by jmacura** [comment]
So the goal of this feature request is to allow the upload of such zip files via our UI ("add-data" panel)?

**Comment by fzadrazil** [comment]
> So the goal of this feature request is to allow the upload of such zip files via our UI ("add-data" panel)?

Yes, exactly. In the similar manner as single rasters. The only difference is in the extra regex input parameter.

**Comment by jmacura** [comment]
> > So the goal of this feature request is to allow the upload of such zip files via our UI ("add-data" panel)?
> 
> Yes, exactly. In the similar manner as single rasters. The only difference is in the extra regex input parameter.

OK. Are imagining that the regex will be input as a plain string or that we prepare a UI for its construction from a series of dialogs?

**Comment by fzadrazil** [comment]
User should be ideally able to set it in some dialog where s/he will select the portion of file name and specify the format string. TC Multi-rename tool could be an inspiration? 
![image](https://user-images.githubusercontent.com/48727502/215551629-6c261dcd-2e11-467e-8533-9a82082147aa.png)
Let's settle this in person on Wednesday.
